.github/workflows/ci.yml

```diff
@@ -36,8 +36,9 @@ jobs:
     - uses: actions/checkout@v4
     - name: install clang repo
       run: |
+        ubuntu_codename=`sed 's/^ *UBUNTU_CODENAME= *\([^ ]*\).*$/\1/p; d;' /etc/os-release`
```

Comment:
Restrict the codename regexp to allow alphanumeric characters and hyphen only. Same in the other locations.

Reply:
I thought the same thing just after I made the PR. The "everything but whitespace" set might be too loose. I narrowed the set to only lowercase alphabets.